### DISCUSSION TITLE
[BUG]Unable to view version information

### DISCUSSION THREAD

**Discussion by winicer** [Chia-Network/chia-blockchain#3498] [Support]
chia-blockchain for Ubuntu
Unable to view version information
![03122927](https://user-images.githubusercontent.com/29593322/116841540-1c574b00-ac0c-11eb-9f3e-e0e60fd197ad.png)

**Comment by sargonas** [comment]
By chance did you install via the "experimental GUI installer" for Ubuntu? I've seen multiple reports of the Version being like this in that mode.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
